assets/css/translate.scss

```
@@ -0,0 +1,40 @@
+// Translation results of " Section x" in articles
+// Autor: Olsza
+// License: CC-BY-4.0
+
+// Enter the locale code and the translation of "Section" in the language.
+// The country code should be the same as the value of the lang attribute in html (use "-" instead of "_")
+$section_translation: (
+        "de": "",
+        "el": "",
+        "en": "section",
+        "es": "sección",
+        "fa": "",
+        "fr": "partie",
```

```suggestion
        "en": "Section",
        "es": "Sección",
        "fa": "",
        "fr": "Partie",
```
The current 'Section' start with a capital letter so these should as well, right?

---

Thx :+1: